config/jobs-chromeos.yaml

```diff
@@ -484,6 +484,17 @@ _anchors:
         - next
         - collabora-chromeos-kernel
 
+  h26forge-debian: &h26forge-debian-job
+    template: 'h26forge-debian.jinja2'
```

Comment:
I was unable to locate this template in [config/runtime](https://github.com/kernelci/kernelci-pipeline/tree/main/config/runtime).

Will there be [a lot of setup](https://github.com/kernelci/kernelci-pipeline/blob/main/config/runtime/fluster-chromeos.jinja2) in the new template? Maybe `generic.jinja2` will be sufficient?